lib/CDVTestLocal.js

```diff
@@ -42,11 +31,6 @@ export default class CDVTestLocal extends CDVTest {
     if (appiumVersion) {
       logger.echo(`Appium installed, appium version: ${appiumVersion}`);
     }
-    // check appium is available
-    if (!this.isAppiumAvailable()) {
-      return new Error('Appium is not running or available, run appium &');
-    }
-    logger.ok('Checking appium is running & available');
     return null;
```

Comment:
@vrunoa because the node_modules folder is not published in the package anymore. And I don't have the time to fix the `appium-running` package. 
The code to start the check was very weird, because of the direct reference of the node_modules folder.

I start appium in any case before starting the tests. I would suggest just to add that to the CDVTestLocal instead of doing an async request to the appium server.

I think I can do that tomorrow.